common-lib/src/test/scala/com/gu/mediaservice/lib/collections/CollectionManagerTest.scala

```diff
@@ -0,0 +1,27 @@
+package com.gu.mediaservice.model
+
+import com.gu.mediaservice.lib.collections.CollectionsManager
+import org.scalatest.{FunSpec, Matchers}
+
+class CollectionManagerTest extends FunSpec with Matchers {
```

Comment:
:boar: should this be CollectionsManagerTest? There isn't a CollectionManager anymore?